sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -124,11 +117,11 @@ export interface CommonGramTokenFilter {
 }
 
 // @public
-export type ComplexDataType = "Edm.ComplexType" | "Collection(Edm.ComplexType)";
+export type ComplexDataType = "Edm.ComplexType";
```

Comment:
did we intend to remove the collection type here?

Reply:
This is the result of the latest PR: https://github.com/Azure/azure-rest-api-specs/pull/9301/files#diff-a01e10da9dba0149960db35a9d106976L2632. As a result the collection is gone from the generated code. As a result, I removed it from the custom code too